app/javascript/components/__snapshots__/widget-display.test.jsx.snap

```diff
@@ -0,0 +1,73 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`DisconnectedDisplay component renders correctly 1`] = `
```

Comment:
Why add the '1' to the end of the test description test?

Reply:
The description (in the snapshot test implementation file) doesn't have a 1 at the end of it. This is an auto generated file, so there is stuff happening behind the scenes that I can't really speak to.